src/logic/safe/utils/safeStorage.ts

```diff
@@ -6,23 +6,16 @@ export const DEFAULT_SAFE_KEY = 'DEFAULT_SAFE'
 
 type StoredSafes = Record<string, SafeRecordProps>
 
-export const loadStoredSafes = async (): Promise<StoredSafes | undefined> => {
-  const safes = await loadFromStorage<StoredSafes>(SAFES_KEY)
-
-  return safes
+export const loadStoredSafes = (): Promise<StoredSafes | undefined> => {
```

Comment:
If this function returns a Promise, it should still be defined as `async`

Reply:
Why?

Reply:
This is the same as

```
function loadJson(url) {
  return fetch(url)
    .then(response => {
      if (response.status == 200) {
        return response.json();
      } else {
        throw new Error(response.status);
      }
    })
}

loadJson('no-such-user.json') // (3)
  .catch(alert); // Error: 404

```

The code is totally valid

Reply:
Yes, messed up with the diff, sorry :+1: